src/components/relationships-panel/RelationshipsPanel.vue

```diff
@@ -103,7 +103,6 @@ export default class RelationshipsPanel extends Vue {
 
 .relationships-vcard {
   font-size: 1rem;
-  background-color: rgba(0, 0, 0, 0.06);
```

Comment:
Removing this means the parent will have to reach into this component (using deep) to set this block's colour. That's generally frowned upon. Instead, add a prop (and default it as needed) to conditionally set this.

Reply:
What about using CSS?  Can't I theme the `.relationship-vcard` class from the parent?  Wouldn't that be more flexible?  I'm happy to pass in a prop if that's the way you'd prefer to do it.

Reply:
`:deep()` is a CSS selector used for reaching into a sub-component (possibly Shadow DOM).

I think it's a bad idea to reach into components to style them. What do you think?

Reply:
PS Maybe rename that class now? relationship-wrapper or relationship-content or something

Reply:
I'll go with your suggestion.  Changes coming up in just a sec ...